.gitignore

```diff
@@ -1,3 +1,4 @@
-/vendor/
-/build/
+vendor/
+build/
```

Comment:
We only want to exclude directories in the project root named vendor or build

Reply:
Sorry, fixed